src/Products/Viewer/Controllers/ViewerApiController.cs

```diff
@@ -563,42 +530,43 @@ private static int GetNewAngleValue(int currentAngle, int postedAngle)
         /// <param name="postedData">Posted data with document guid.</param>
         /// <param name="loadAllPages">Flag to load all pages.</param>
         /// <returns>Document pages data, dimensions and rotation angles.</returns>
-        private static LoadDocumentEntity GetDocumentPages(PostedDataEntity postedData, bool loadAllPages)
+        private static LoadDocumentEntity GetDocumentPages(PostedDataEntity postedData, bool loadAllPages, IFileWrapper fileWrapper)
         {
             // get/set parameters
             string documentGuid = postedData.guid;
             string password = string.IsNullOrEmpty(postedData.password) ? null : postedData.password;
 
-            var fileFolderName = Path.GetFileName(documentGuid).Replace(".", "_");
+            var fileFolderName = fileWrapper.GetFileFolderName(documentGuid);
             string fileCacheSubFolder = Path.Combine(cachePath, fileFolderName);
 
-            if (!File.Exists(documentGuid))
-            {
-                throw new GroupDocsViewerException("File not found.");
-            }
+            // TODO: do we need this?
+            //if (!File.Exists(documentGuid))
+            //{
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=6117781)